src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/io/AprilTagLimelightIO.java

```diff
@@ -0,0 +1,49 @@
+package frc.trigon.robot.poseestimation.apriltagcamera.io;
+
+import frc.trigon.robot.constants.FieldConstants;
+import frc.trigon.robot.poseestimation.apriltagcamera.AprilTagCameraIO;
+import frc.trigon.robot.poseestimation.apriltagcamera.RobotPoseSourceInputsAutoLogged;
+import org.trigon.utilities.LimelightHelpers;
+
+public class AprilTagLimelightIO extends AprilTagCameraIO {
+    private final String hostname;
+
+    public AprilTagLimelightIO(String hostname) {
+        this.hostname = hostname;
+    }
+
+    @Override
+    public void updateInputs(RobotPoseSourceInputsAutoLogged inputs) {
```

Comment:
Extarct into two functions like in PV's impl
This should generally look the same between the two
Maybe even create protected functions in AprilTagCameraIO